src/components/LoginForm/index.js

```diff
@@ -0,0 +1,127 @@
+          <p>Remember me</P>
+        </div>
+        <a href="#" class="forgot-password">Esqueceu a senha?</a>
+      </div>
+    </form>
+
+    <p class="login-form-container__signup">Não tem uma conta? 
+      <a href="#" class="signup-link">Inscreva-se</a>
+    </p>
+
+    <div class="login-form-container__separator"></div>
+
+    <div data-select="provider-container" class="login-form-container__provider-container">
+      <button class="provider-button"><img src="${googleIcon}" class="icon"/ >Google</button>
+      <button class="provider-button"><img src="${facebookIcon}" class="icon"/ >Facebook</button>
+    </div>
+  </div>
+`;
+
+export default function LoginForm() {
+  Component.call(this, { html });
+  const $loginForm = this.selected.get('login-form');
+  const $inputContainer = this.selected.get('input-container');
+  const $rememberOption = this.selected.get('remember-option');
+  const $errorMessage = this.selected.get('error-message');
+
+  const emailInput = new TextInput({ placeholder: 'E-mail' });
+  const passwordInput = new TextInput({
+    placeholder: 'Senha',
+    assetUrl: eyeSlashIcon,
+    assetPosition: 'suffix',
+  });
+  const toggle = new Toggle({ checked: false });
+  const buttonSubmit = new Button({
+    text: 'Entrar',
+    isFullWidth: true,
+    isDisabled: true,
+  });
+
+  emailInput.mount($inputContainer);
+  passwordInput.mount($inputContainer);
+  buttonSubmit.mount($loginForm);
+  toggle.mount($rememberOption);
+
+  emailInput.selected.get('input-text').type = 'email';
+  emailInput.selected.get('input-text').id = 'email';
+  passwordInput.selected.get('input-text').type = 'password';
+
+  const checkInputs = () => {
+    const emailValue = emailInput.selected.get('input-text').value;
+    const passwordValue = passwordInput.selected.get('input-text').value;
+    if (emailValue.trim() !== '' && passwordValue !== '') {
+      buttonSubmit.enable();
+      return;
+    }
+    buttonSubmit.disable();
+  };
+
+  emailInput.selected.get('input-text').addEventListener('input', checkInputs);
+  passwordInput.selected
+    .get('input-text')
+    .addEventListener('input', checkInputs);
+
+  buttonSubmit.listen('click', () => {
+    const emailValue = emailInput.selected.get('input-text').value;
+    const passwordValue = passwordInput.selected.get('input-text').value;
+
+    if (!this.validateEmail(emailValue)) {
+      $errorMessage.style.display = 'block';
+      $errorMessage.innerText = 'E-mail inválido';
+      emailInput.inputError();
+      return;
+    }
+
+    if (!this.validatePassword(passwordValue)) {
+      $errorMessage.style.display = 'block';
+      $errorMessage.innerText =
+        'Senha inválida. Sua senha deve conter no mínimo 10 caracteres, incluindo pelo menos um caractere especial e uma letra maiúscula.';
+      passwordInput.inputError();
+      return;
+    }
+
+    $errorMessage.style.display = 'none';
+    this.login();

```

Comment:
Aqui por fim, acho que poderiamos ter uma variavel de controle nas validações, algo tipo `validFields` pra sconder o erro e fazer o login
```suggestion
if (validFields)
    $errorMessage.style.display = 'none';
    this.login();
}
```